code/modules/scavstation_shared_content/species/baxxid.dm

```diff
@@ -39,6 +46,8 @@
 		if(first_char != lowertext(first_char))
 			hnnn = uppertext(capitalize(hnnn))
 		. = "[hnnn][uppertext(.)]"
+	if(autohiss_exempt && (lang.name in autohiss_exempt))
```

Comment:
This should really be at the top of the proc, before the message is processed.

Reply:
Moved autohiss in baxxid.dm (lines 39 & 40)